docker/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# syntax=docker/dockerfile:1.4
+
+ARG BASE_IMAGE
+
+FROM ${BASE_IMAGE}
+
+ARG IMAGE_ENTRYPOINT="exec java -Dlog4j2.debug=false -Dlog4j.configurationFile=log4j2-console.xml -jar /opt/override/crypto.jar"
+ARG BUILD_PATH="/applications/workers/release/crypto-worker/build/tmp/publishOSGiImage/containerization/corda-crypto-worker-5.0.0.0-SNAPSHOT.jar"
```

Comment:
`crypto-worker` in Plugin Host?
Was it intentional or a copy/paste mistake?

Reply:
Yeah it's a copy paste. It is a default value that gets overwritten every time. Might as well set it to null to be honest

Reply:
removed default params entirely